src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IIsPatternExpression.cs

```diff
@@ -1455,6 +1457,12 @@ class D
 ";
 
             VerifyOperationTreeForTest<IsPatternExpressionSyntax>(compilation, expectedOperationTree);
+
+            var tree = compilation.SyntaxTrees[0];
+            var isPattern = tree.GetRoot().DescendantNodes().OfType<IsPatternExpressionSyntax>().Single();
+            var model = compilation.GetSemanticModel(tree);
+            var instanceReceiver = model.GetOperation(isPattern).Descendants().OfType<IInstanceReferenceOperation>().Single();
+            TestOperationVisitor.Singleton.Visit(instanceReceiver);
```

Comment:
Aleksey commented:

> It is not clear to me what is this doing which VerifyOperationTreeForTest wouldn't already do. Could you please elaborate?

Reply:
`VerifyOperationTreeForTest` does not call the `TestOperationVisitor` on operation nodes. Perhaps it should, but that is what this is accomplishing.

---
In reply to: [551558624](https://github.com/dotnet/roslyn/pull/50111#discussion_r551558624) [](ancestors = 551558624)